main.tf

```diff
@@ -743,3 +743,33 @@ module "cbr_rule" {
   }]
   operations = var.cbr_rules[count.index].operations == null ? local.default_operations : var.cbr_rules[count.index].operations
 }
+
+##############################################################
+# Ingress Secrets Manager Integration
+##############################################################
+
+resource "ibm_iam_authorization_policy" "ocp_secrets_manager_iam_auth_policy" {
+  count                       = var.enable_secrets_manager_integration ? 1 : 0
```

Comment:
you need to provide the ability for users to opt out of this incase they are managing s2s auth policies some other way. Add a new variable `skip_secrets_manager_iam_auth_policy` and include it in the count logic here. Should be added to the DA too